src/views/SemesterAdminTable.vue

```diff
@@ -0,0 +1,374 @@
+          </router-link>
+        </template>
+        <template v-slot:foot(proposal)>
+          <span class="font-weight-bold">Total: {{ filteredTimeallocations.length }}</span>
+        </template>
+        <template v-slot:cell(pi)="data">
+          <div v-for="(pi, idx) in data.item.proposal.pis" :key="pi.first_name + pi.last_name + idx">{{ pi.first_name }} {{ pi.last_name }}</div>
+        </template>
+        <template v-slot:cell(std_allocation)="data">
+          {{ data.item.std_allocation | formatFloat(1) }}
+        </template>
+        <template v-slot:foot(std_allocation)>
+          <span class="font-weight-bold">{{ sumFormatter('std_allocation') }}</span>
+        </template>
+        <template v-slot:cell(std_time_used)="data">
+          {{ data.item.std_time_used | formatFloat(1) }}
+        </template>
+        <template v-slot:foot(std_time_used)>
+          <span class="font-weight-bold">{{ sumFormatter('std_time_used') }}</span>
+        </template>
+        <template v-slot:cell(rr_allocation)="data">
+          {{ data.item.rr_allocation | formatFloat(1) }}
+        </template>
+        <template v-slot:foot(rr_allocation)>
+          <span class="font-weight-bold">{{ sumFormatter('rr_allocation') }}</span>
+        </template>
+        <template v-slot:cell(rr_time_used)="data">
+          {{ data.item.rr_time_used | formatFloat(1) }}
+        </template>
+        <template v-slot:foot(rr_time_used)>
+          <span class="font-weight-bold">{{ sumFormatter('rr_time_used') }}</span>
+        </template>
+        <template v-slot:cell(tc_allocation)="data">
+          {{ data.item.tc_allocation | formatFloat(1) }}
+        </template>
+        <template v-slot:foot(tc_allocation)>
+          <span class="font-weight-bold">{{ sumFormatter('tc_allocation') }}</span>
+        </template>
+        <template v-slot:cell(tc_time_used)="data">
+          {{ data.item.tc_time_used | formatFloat(1) }}
+        </template>
+        <template v-slot:foot(tc_time_used)>
+          <span class="font-weight-bold">{{ sumFormatter('tc_time_used') }}</span>
+        </template>
+        <template v-slot:cell(ipp_limit)="data">
+          {{ data.item.ipp_limit | formatFloat(1) }}
+        </template>
+        <template v-slot:foot(ipp_limit)>
+          <span class="font-weight-bold">{{ sumFormatter('ipp_limit') }}</span>
+        </template>
+        <template v-slot:cell(ipp_time_available)="data">
+          {{ data.item.ipp_time_available | formatFloat(1) }}
+        </template>
+        <template v-slot:foot(ipp_time_available)>
+          <span class="font-weight-bold">{{ sumFormatter('ipp_time_available') }}</span>
+        </template>
+        <template v-slot:foot()>
+          <br />
+        </template>
+      </b-table>
+    </data-loader>
+    <!-- These are included for downloading the table data-->
+    <script src="https://cdn.lco.global/script/tableExport.min.js" type="application/javascript"></script>
```

Comment:
This seems hacky, but I couldn't find an npm package for this lib. That, and this file is being pulled from our CDN, so I didn't want to change it.

Reply:
hmm strange it seems weird that we would have written this...  I bet we just put it on our cdn to be safe.